nixos/modules/config/pulseaudio.nix

```diff
@@ -160,6 +167,31 @@ in {
             if activated.
           '';
         };
+
+        resample-method = mkOption {
+          type = types.str;
```

Comment:
that’s why I propose just having a `config` and not copy items with potentially out-of-date defaults when upstream decides to change them.